README.md

```diff
@@ -155,7 +155,7 @@ This section contains information for building this project.
 
 2. Build Stateful Functions Docker image: This step requires that you've already compiled artifacts from the source code.
   ```
-  $ ./tools/docker/build-distribution.sh
+  $ ./tools/docker/build-stateful-functions.sh
```

Comment:
was this done by accident?

Reply:
That's actually the name of the script: `./tools/docker/build-stateful-functions.sh`. The README in the docker folder has the right name but the top-level README had `./tools/docker/build-distribution.sh`.